README.md

```diff
@@ -68,6 +67,18 @@ workers implement the Remote Worker API.
 * [Scoot](https://github.com/twitter/scoot) (open source)
 * [Turbo Cache](https://github.com/allada/turbo-cache) (open source)
 
+### Workers
+Servers are responsible for managing a pool of workers.
+The Remote Worker API defines a reference protocol for worker and server
+communication, although, adhering to this protocol is not a requirement.
+Typically, workers are domain specific, optimized for custom scheduling
+protocols defined by the server implementation.
+
+* [Remote Worker API](https://goo.gl/oojM5H)
+* [Buildfarn](https://bazelbuild.github.io/bazel-buildfarm/docs/architecture/queues/)
+* [Buildbarn](https://nluug.nl/bestanden/presentaties/2023-05-11-ed-schouten-buildbarn-a-distributed-build-cluster.pdf)
```

Comment:
Please link to this instead: https://github.com/buildbarn/bb-remote-execution/blob/master/pkg/proto/remoteworker/remoteworker.proto